tests/utils/trade.py

```diff
@@ -112,11 +112,13 @@ def get_order(self, robot_index=1, first_encounter=False):
         self.response = self.client.get(path + params, **headers)
 
     @patch("api.tasks.send_notification.delay", send_notification)
-    def cancel_order(self, robot_index=1):
+    def cancel_order(self, robot_index=1, cancel_status=None):
         path = reverse("order")
         params = f"?order_id={self.order_id}"
         headers = self.get_robot_auth(robot_index)
         body = {"action": "cancel"}
+        if cancel_status is not None:
+            body.update({"cancel_status": cancel_status})
```

Comment:
A little bit cumbersome. We can leave this simple as `body = {"action": "cancel", "cancel_status": cancel_status}` without need for `if` statement right ?

Reply:
It is a small thing, but in this way the test better fits with how the api is actually called by clients in mainnet. The client either specifies a non `None` `cancel_status`, or it does not specifiy `cancel_status` all together, it is quite rare that it will specify `cancel_status` as `None`. It is really a minor thing, can also be changed with the way you specified.

Reply:
I used the if condition because of what Jerry mentions, but that does not affect these tests, so I will replace it with your suggestion instead

Reply:
Have no preference really, both views are valid now that I read the thinking behind.